warrior/WarriorCore/project_driver.py

```diff
@@ -241,42 +291,118 @@ def execute_project(project_filepath, auto_defects, jiraproj, res_startdir, logs
             if goto_testsuite and goto_testsuite == str(suite_cntr):
                 goto_testsuite = False
             elif goto_testsuite and goto_testsuite != str(suite_cntr):
-                data_repository['testsuite_%d_result'%suite_cntr] = "ERROR"
+                data_repository['testsuite_{}_result'.format(suite_cntr)] = "ERROR"
                 continue
 
-        goto_testsuite_num = onerror_driver.main(testsuite, project_error_action, project_error_value)
-        if goto_testsuite_num == False:
+        goto_testsuite_num = onerror_driver.main(testsuite,
+                                                 project_error_action,
+                                                 project_error_value)
+        if goto_testsuite_num is False:
             onerror = "Next"
-        elif goto_testsuite_num == "ABORT":
+        elif goto_testsuite_num is "ABORT":
```

Comment:
https://stackoverflow.com/questions/1504717/why-does-comparing-strings-in-python-using-either-or-is-sometimes-produce
Please refer to the above link, try avoid using is in this kind of string comparison